app/styles/app/modules/request/configs.scss

```diff
@@ -3,6 +3,18 @@
     border: 1px solid $dry-cement;
     margin-bottom: 1em;
   }
+
+  h4 {
```

Comment:
ok? i'm curious why :) it's always been my habit to try and keep things simple and puristic, but i can see how maybe that is a little out of date.

i will of course change this.

Reply:
Eh, it's not really a big deal, you can leave this as-is if you want. It's actually a bit of a debate in the frontend world about what are the best ways to use the cascade. I'll gather some info about it and add another comment :)

Reply:
i see :) it's alright ...

i believe i've addressed this in https://github.com/travis-ci/travis-web/commit/5003d2b2e8ab86a2dbb2819bf4015c6e3255d755

Reply:
Looks good to me :+1: Thanks!

Reply:
@svenfuchs frontend apps are getting more and more complex, and stylesheets are growing as well. And in such a complex apps it's hard to manage cascading, especially with componential approach. Using just tag selectors you define styles for all nested elements of this tag, so if markup would change and someone would add another nested element with the same tag, your style will apply to it as well. And It's sometimes very hard to overwrite. With complexity growing, it could quickly become a mess. Mainly because of this, currently there's a trend to avoid cascading at all whenever possible (there are post-css modules that add hashes to classes to make them unique), or at least scope styles to specific components (BEM for example). But using classes is already helping a lot in this sense, so it's basically like a rule of thumb.